components/QuizCardPage/NextButton/index.test.js

```diff
@@ -1,16 +1,28 @@
-import { StyledButtonNext } from ".";
-import { render, fireEvent, screen } from "@testing-library/react";
+import { render, screen } from "@testing-library/react";
+import NextButton from ".";
 
-describe("StyledButtonNext component", () => {
-  it("should call onClick to move to the next page", () => {
-    const onClick = jest.fn();
-    const moveNextPage = "Next";
+jest.mock("next/router", () => ({
+  useRouter: () => ({
+    push: jest.fn(),
+  }),
+}));
 
-    render(
-      <StyledButtonNext onClick={onClick}>{moveNextPage}</StyledButtonNext>
-    );
-
-    fireEvent.click(screen.getByText("Next"));
-    expect(onClick).toHaveBeenCalledTimes(1);
-  });
+test("renders NextButton component without crashing", async () => {
```

Comment:
This description does not fit your `expect().not.toBeInTheDocument()`: you are here testing for the NextButton to be not rendered.

Reply:
at the beginning because I had set disabled to false, but now I've set it to true and now I'm testing whether the button is there